changelogs/fragments/ini_extension.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ini inventory - The plugin now has an 'allowed_extensions' option to specify which extensions the plugin will consider as an inventory source. It defaults to '.ini' and '' (no extension), which should cover most usage. For the previous behavior of matching files regardless of extension, set it to an empty list. 
```

Comment:
Between this changelog fragment, the options documentation, the implementation, and the tests, I can't tell what the desired behavior is supposed to be after this change, or what it will be once the deprecation period is over. Can you clarify?